reflex/constants/base.py

```diff
@@ -173,3 +173,4 @@ class Ping(SimpleNamespace):
 # Testing variables.
 # Testing os env set by pytest when running a test case.
 PYTEST_CURRENT_TEST = "PYTEST_CURRENT_TEST"
+RELOAD_CONFIG = "RELOAD_CONFIG"
```

Comment:
```suggestion
RELOAD_CONFIG = "__REFLEX_RELOAD_CONFIG"
```

i think we should namespace this magic var, like `SKIP_COMPILE_ENV_VAR` so it is less likely to be accidentally set